Sources/ScipioKit/Producer/Compiler.swift

```diff
@@ -44,6 +45,13 @@ struct Compiler<E: Executor> {
             debugSymbolPaths = nil
         }
 
+        let frameworkName = target.xcFrameworkName
+        let outputXCFrameworkPath = outputDirectory.appendingPathComponent(frameworkName)
+        if fileSystem.exists(outputXCFrameworkPath) && force {
+            logger.info("💥 Delete \(frameworkName)", metadata: .color(.red))
+            try fileSystem.removeFileTree(at: outputXCFrameworkPath)
+        }
+
         try await xcodebuild.createXCFramework(
```

Comment:
Isn't it still overwrite xcframework even if we don't have above new code?

Reply:
No. `xcodebuild` raises an error saying it can't overwrite existing XCFramework.